applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -72,10 +82,17 @@ The possible cases that impact how the host to which the nRF desktop dongle is c
 When reports are forwarded to the :ref:`nrf_desktop_usb_state`, each HID-class USB device is treated independently.
 This means that the |hid_forward| can concurrently send reports to every available HID-class USB device, if a report from the linked peripheral is available.
 
+When received by the module, the HID boot protocol reports come with the report ID assigned to :c:enum:`REPORT_ID_RESERVED`.
+This ID is later changed by |hid_forward| to either :c:enum:`REPORT_ID_BOOT_KEYBOARD` or :c:enum:`REPORT_ID_BOOT_MOUSE`.
+The choice depends on :c:struct:`bt_hogp` objects.
```

Comment:
depends on fact, if the Bluetooth device is boot mouse or keyboard ? I am not sure if we should refer to internal data structures here.